carbon-design-system/carbon
# chore(styles): remove v10 deprecated styles for v11

Closes #8419 

#### Changelog

**Changed**

- Updated the migration guide with some additional info related to a couple of these removals

**Removed**

- Remove deprecated styles

#### Testing / Reviewing

- View stories for all impacted components, ensure no regressions
  - [ ] Grid
  - [ ] ContentSwitcher
  - [ ] TextInput
  - [ ] Type Reset
- Ensure the migration guide changes make sense
